packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -706,6 +706,47 @@ export default abstract class BaseCall implements IWebRTCCall {
     toggleAudioTracks(this.options.remoteStream);
   }
 
+  setAudioBandwidth(min: number, max: number) {
+    const { instance } = this.peer;
+    const sender = instance
+        .getSenders()
+        .find(({ track: { kind } }: RTCRtpSender) => kind === 'audio');
+
+    if (sender) {
+
+        let p = sender.getParameters();
+        const parameters = p as RTCRtpSendParameters;
+        if (!parameters.encodings) {
+          parameters.encodings = [{ rid : 'h' }];
+        }
+        console.log("Parameters: ", parameters);
+
+        if (min) {
+            console.log("Setting min audio bandwidth to: ", min, " [kbps]");
+            //parameters.encodings[0].minBitrate = min * 1024;
```

Comment:
is this should exist?

Reply:
Yeah, for now min bitrate is not really set here, cause RTCRtpSendParameters doesn't have this property (only maxBitrate), but in another commit I want to add static bandwidth configuration - that happens at call setup. That one will use SDP munging (SDP hack) and will be executed from here.